src/inmanta/app.py

```diff
@@ -767,9 +778,7 @@ def _get_watched_file_handler(options: argparse.Namespace) -> logging.handlers.W
     return file_handler
 
 
-def _convert_to_log_level(level: int) -> int:
```

Comment:
For me, it took some time to interpret this method and the method below this method. This is not an issue introduced by your changes, but we should grasp the opportunity to improve it now that we are working on this part of the code base. It would be good to add a docstring to both methods, describing what these methods do. We should definitely mention that:

* `_convert_cli_log_level()`: Converts the number of `-v`'s passed on the CLI to the corresponding Inmanta log level.
* `_convert_to_log_level()`: This method converts the Inmanta log level to the Python log level. I think we could just as well rename this method to `convert_inmanta_log_level_to_python_log_level()`.